components/renku_data_services/authn/dummy.py

```diff
@@ -48,11 +48,21 @@ async def authenticate(access_token: str, request: Request) -> base_models.APIUs
         except:  # noqa: E722 # nosec: B110
             pass
 
-        is_set = bool(user_props.get("id") or user_props.get("name") or user_props.get("is_admin") is not None)
+        is_set = bool(
+            user_props.get("id")
+            or user_props.get("name")
+            or user_props.get("is_admin") is not None
+            or user_props.get("first_name")
+            or user_props.get("last_name")
+            or user_props.get("email")
+        )
 
         return base_models.APIUser(
             is_admin=user_props.get("is_admin", False),  # type: ignore[arg-type]
             id=user_props.get("id") if user_props.get("id") else "some-id" if is_set else None,
             access_token=access_token,
+            first_name=user_props.get("first_name") if user_props.get("first_name") else "John" if is_set else None,
+            last_name=user_props.get("last_name") if user_props.get("last_name") else "Doe" if is_set else None,
+            email=user_props.get("email") if user_props.get("email") else "john.doe@gmail.com" if is_set else None,
```

Comment:
I think you can just put "John" as the second parameter to `get` to get rid of one if,else. same on the other lines. and in general if we need multiple if/else in a line it's nice to add brackets to make the order clear

Reply:
Yeah that is right. I kinda left this as is before because it is only in the dummy stuff used only for testing. But still it is nice to have it cleaner like this.